source/includes/databases-collections/databases-collections.php

```diff
@@ -0,0 +1,93 @@
+<?php
+require 'vendor/autoload.php';
+
+use MongoDB\BSON\Document;
+use MongoDB\Driver\ReadConcern;
+use MongoDB\Driver\ReadPreference;
+use MongoDB\Driver\WriteConcern;
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('Set the MONGODB_URI variable to your Atlas URI that connects to the sample dataset');
```

Comment:
"that connects to the sample dataset" doesn't seem relevant for this file.

Reply:
Yeah users don't see this, but I reworded slightly